tests/integration/cluster.rs

```diff
@@ -267,6 +268,28 @@ pub mod pubsub {
 
 }
 
+
+pub mod other {
+  use super::*;
+
+  #[test]
+  fn it_should_memory_usage_simple_db() {
+    let config = RedisConfig::default_centralized();
+    utils::setup_test_client(config, TIMER.clone(), |client| {
+      other_tests::memoryusage::should_memory_usage_simple(client)
+    })
+  }
+
+  #[test]
+  fn it_should_memory_usage_hset_db() {
+    let config = RedisConfig::default_centralized();
```

Comment:
Shouldn't this and the one above be:
```suggestion
    let config = RedisConfig::default_clustered();
```

Reply:
ooops, thanks for catching that!